urbandict-search/app.py

```diff
@@ -27,7 +27,7 @@ def print_topk(resp, word):
             score = match.score.value
             if score <= 0.0:
                 continue
-            doc = match.meta_info.decode()
+            doc = match.chunks[0].text
             word, word_def = doc.split('+-=', maxsplit=1)
```

Comment:
why not directly match.text?

Reply:
It seems match does not has a attribute of text or match_doc

Reply:
Weird... ok. It should be taken a look afterwards ... Isn't it weird @hanxiao

Reply:
This example does not have a crafter, maybe the problem is from here?

Reply:
This part has some bug, in `match`, the whole matched document is cut into chunks, i.e.
```
matches {
  id: 46
  chunks {
    id: 1569659874
    weight: 1.0
    length: 5
    mime_type: "text/plain"
    text: "dissed+-=1."
    level_depth: 1
    parent_id: 46
    location: 0
    location: 11
    embedding {
    }
  }
  chunks {
    id: 3978505557
    weight: 1.0
    length: 5
    mime_type: "text/plain"
    text: "dissing someone is showing disrespect to them"
    level_depth: 1
    parent_id: 46
    location: 11
    location: 57
    offset: 1
    embedding {
    }
  }
  chunks {
    id: 2304622410
    weight: 1.0
    length: 5
    mime_type: "text/plain"
    text: "being dissed is the act of being disconnected,"
    level_depth: 1
    parent_id: 46
    location: 63
    location: 110
    offset: 3
    embedding {
    }
  }
  chunks {
    id: 1326253678
    weight: 1.0
    length: 5
    mime_type: "text/plain"
    text: "by voice or by modem,"
    level_depth: 1
    parent_id: 46
    location: 110
    location: 132
    offset: 4
    embedding {
    }
  }
  chunks {
    id: 2465893560
    weight: 1.0
    length: 5
    mime_type: "text/plain"
    text: "from another party"
    level_depth: 1
    parent_id: 46
    location: 132
    location: 151
    offset: 5
    embedding {
    }
  }
  weight: 1.0
  length: 16
  mime_type: "text/plain"
  score {
    value: 0.0012319303350523114
    op_name: "NumpyIndexer"
    ref_id: 1
  }
}
```

Reply:
Ok, we will take a look at it.